jarbas/settings.py

```diff
@@ -23,10 +24,10 @@
 # See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = config('SECRET_KEY', default='my-secret')
+SECRET_KEY = config('SECRET_KEY', default=secrets.token_hex())
```

Comment:
Generating a random value here might lead the user to problems in the future (for example, if using authentication module, this might break password hashes because the secret key will change). If you need to “force” user to use a random string the best choice is to remove the `default` argument and leave it as mandatory to use whatever is in the `.env` file.